nltk/corpus/reader/conll.py

```diff
@@ -58,7 +59,7 @@ class ConllCorpusReader(CorpusReader):
     CHUNK = 'chunk'   #: column type for chunk structures
     NE = 'ne'         #: column type for named entities
     SRL = 'srl'       #: column type for semantic role labels
-    IGNORE = 'ignore' #: column type for column that should be ignored
+    IGNORE = 'ignore'  # : column type for column that should be ignored
```

Comment:
It's a shame the pep8 processor isn't able to detect a block of comments and line them up.